implementation/observation-otel-bridge/src/main/java/io/smallrye/opentelemetry/instrumentation/observation/handler/PropagatingReceiverTracingObservationHandler.java

```diff
@@ -56,6 +56,7 @@ public String get(C carrier, String key) {
                 return getter.get(carrier, key);
             }
         });
+        // extracted.makeCurrent(); // this actually fixes the baggage test
```

Comment:
Is this ok?

Reply:
Basically, if you uncomment, baggage propagation will be fixed, is there for reference... We can remove these comments but I will need to document these things somewhere, maybe on a README... What do you think?